.github/workflows/build.yml

```diff
@@ -11,18 +11,37 @@ jobs:
     name: Goreleaser Dry
     runs-on: ubuntu-latest
     steps:
+      - name: Check out the code
+        uses: actions/checkout@v3
+
       - name: Set up Go
         uses: actions/setup-go@v3
         with:
           go-version: ^1.18
 
-      - name: Install llvm
-        run: sudo apt-get install llvm libelf-dev
+      - name: Set up Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          components: rustfmt, clippy
 
-      - name: Checkout
-        uses: actions/checkout@v3
+      - name: Set up Rust Nightly
+        uses: actions-rs/toolchain@v1
```

Comment:
This is so we install some components from 'stable' but rustc from nightly, right? In that case we should make sure we run something similar locally

Reply:
We have a make action to install these in a similar manner under /bpf